src/ui/MessageInput/__tests__/MessageInput.spec.js

```diff
@@ -45,28 +46,75 @@ describe('ui/MessageInput', () => {
     ).toBe(0);
   });
 
-  it.skip('should render send icon if text is present', () => {
-    const component = shallow(
-      <MessageInput
-        onSendMessage={noop}
-        message={{
-          message: 'hello',
-          mentionedMessageTempalte: 'hello'
-        }}
-      />
-    );
+  it('should call sendMessage with valid string', async () => {
+    const onSendMessage = jest.fn();
+    const textRef = { current: { innerText: null } };
+    const mockText = 'Test Value';
+
+    render(<MessageInput onSendMessage={onSendMessage} ref={textRef} />);
+
+    const input = screen.getByRole('textbox');
+    await userEvent.clear(input);
+    await userEvent.type(input, mockText);
+    expect(input.textContent).toBe(mockText);
+
+    fireEvent.keyDown(input, { key: 'Enter' });
+    expect(onSendMessage).toHaveBeenCalled();
```

Comment:
nit: Can we do a `toHaveBeenCalledWith` assertion instead?
 https://jestjs.io/docs/expect#tohavebeencalledwitharg1-arg2-

Reply:
Similar for other tests

Reply:
Yeah just did https://github.com/sendbird/sendbird-uikit-react/pull/538/commits/c128ae24d0b4531d61fb8e679e277ceb22e32260